packages/pages/tests/fixtures/template.js

```diff
@@ -0,0 +1,106 @@
+var __create = Object.create;
```

Comment:
this JS file is already transpiled so it'll load but there's no guarantee other js files will be written in a form accepted by default

Reply:
Are you saying to rerun it through the transpilation step?

Reply:
I was just saying that this file doesn't seem like a realistic fixture because the module appears to be already transpiled (unless that was the explicit intention of the test)